java/clients/src/test/resources/reports/ingest/job/standard/detailed/mixedJobs.txt

```diff
@@ -1,54 +1,95 @@
 
 Ingest Job Status Report
 ------------------------
-Details for job $(jobId4):
+Details for job job77777-7777-7777-7777-777777777777:
+State: FINISHED
+Number of input files: 7
+
+Run on task task3333-3333-3333-3333-333333333333
+Start Time: 2022-09-27T13:34:12.001Z
+Start Update Time: 2022-09-27T13:34:12.123Z
+Unknown update type: IngestJobAddedFilesStatus
```

Comment:
This doesnt look right? It looks like its coming from `StandardProcessRunsReporter.printProcessJobRun`, we probably need to refactor this to support more than 1 custom UpdatePrinter (one for validation status, one for added file status)

Reply:
Oops, I forgot that still needed doing. Thanks.

Reply:
Done, thanks.